eventFunctions.js

```diff
@@ -0,0 +1,53 @@
+
+var CarLot = (function(cars) {
+
+
+
+  cars.activateEvents = function(carDetails) {
+    var changeMi;
+    var cards = document.getElementsByClassName("card-block"); //returns each individual card div in an array
+    // console.log(cards);
+    var textEdit = document.getElementById("editText"); //gets id of textbox
+
+    for (var i = 0; i < cards.length; i++) { //loops through array of card divs
+
+      var cardsLooped = cards[i];
+
+      // console.log("",cardsLooped);
+
+        //onClick event on looped cards
+      cardsLooped.addEventListener("click", function(event) {
+        // console.log("logEvents",event );
+        // adds css styling to cards
+        event.currentTarget.classList.toggle("animateClass")
```

Comment:
This line doesn't achieve the intended functionality. Maybe think about adding a function that will satisfy the requirements of passing the car card and color into it, then changing the styles.